Добавить перезапуск упавших тестов

meridian285 opened the pull request:
Добавить класс TestRetry
Добавить метод testRetry в BaseTest

meridian285 committed:
Добавить перезапуск упавших тестов

Добавить класс TestRetry
Добавить метод testRetry в BaseTest

nazarovvi commented on src/test/java/com/way2automation/help/TestRetry.java:
```diff
@@ -0,0 +1,24 @@
+    public boolean retry(ITestResult iTestResult) {
+        // Условие выполняется если тест упал
+        if (!iTestResult.isSuccess()) {
+            int maxCount = 2;
```
Вынеси количество перезапусков в проперти

nazarovvi commented on src/test/java/com/way2automation/help/TestRetry.java:
```diff
@@ -0,0 +1,24 @@
+    @Override
+    public boolean retry(ITestResult iTestResult) {
+        // Условие выполняется если тест упал
+        if (!iTestResult.isSuccess()) {
```
Объедини ифы в один

meridian285 committed:
Добавить application.properties, ReadProperties

значение maxCount для TestRetry берется из application.properties
Добавлена библиотека config

meridian285 committed:
Добавить изменение блока if в TestRetry.retry()

nazarovvi commented:
Можно мержить
